tests/jsTests/indexSpec.js

```diff
@@ -1,5 +1,9 @@
-describe("Tests for show error", function() {
-	var msg = 'this is a test message';
+/**
+ * Suite : Tests for showing error on incorrect field entry
+ * Description : This is a test suite for showing error in case of an invalid field entry
+ */
+describe("Tests for showing error on incorrect field entry", function() {
+	var msg = 'This is a test message for showing error';
```

Comment:
Make this message more specific to the kind of error or is this generic for all types of errors under this suite?